docs/content/cli-commands/npm-install.md

```diff
@@ -63,8 +64,7 @@ after packing it up into a tarball (b).
     With the `--production` flag (or when the `NODE_ENV` environment variable
     is set to `production`), npm will not install modules listed in
     `devDependencies`. To install all modules listed in both `dependencies` 
-    and `devDependencies` when `NODE_ENV` environment variable is set to `production`, 
-    you can use `--production=false`.
+    and `devDependencies` when `NODE_ENV` environment variable is set to `production`, you can use `--production=false`.
```

Comment:
@claudiahdz I think the reason there were line breaks here was to keep the text column count short in a shell output... not sure if removing them will effect that negatively so that command line docs would span the length of the terminal 🤔